opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -779,7 +780,7 @@ def _dissociate_port_from_es(self, port_id, port_mac=None, ess=None):
             es_list = self.es_port_dict.keys()
         else:
             es_list = ess
-        for es in es_list:
+        for es in list(es_list):
```

Comment:
Is this list() still needed?

Reply:
Yes. Without wrapping of list, it gives: "RuntimeError: dictionary changed size during iteration"